### ISSUE TITLE
Question: Are gold drops supposed to be that generous?

### ISSUE THREAD

**Issue by Cryomaniac13** [hengband/hengband#4731]
I'm playing beta22, and I'm seeing that early game gold drops are far greater than usual. Orcs in Orc Camp quest dropped several piles of valuable gems worth hundreds of AU, and I seen a small kobold on dungeon level 1 drop a $1000+ adamantite pile. Is this proper behaviour?

**Comment by dis-** [comment]
Yes, this change is intentionally. We increased gold drops in early games for purchasing scrolls and potions.
(Begginers must buy enough amount of scroll of telepotation and potion of cure critical wounds!)
All gold-drop monsters may have adamantite pile, but the probability is set  quite low. So, simply you were lucky.

**Comment by Cryomaniac13** [comment]
Yes, from my further experience, that was luck on my part. Good to know increased early game money is intentional.
